src/API.js

```diff
@@ -182,7 +182,7 @@ class Connection {
             data.auth_ticket = this.auth_ticket
 
             this.signatureBuilder.setAuthTicket(this.auth_ticket)
-            this.signatureBuilder.setLocation(userObj.latitude, userObj.longitude, userObj.altitude)
+            this.signatureBuilder.setLocation(userObj.latitude, userObj.longitude, userObj.accuracy)

```

Comment:
POGOProtos.Networking.Envelopes.Unknown6 does not exists anymore in node-pogo-protos 1.5.0 it has been replaced with POGOProtos.Networking.Envelopes.RequestEnvelope.PlatformRequest